src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/CountReadsSparkIntegrationTest.java

```diff
@@ -21,22 +21,24 @@ public String getTestedClassName() {
     @DataProvider(name="filenames")
     public Object[][] filenames() {
         return new Object[][]{
-                {"count_reads.sam", null, 8L},
-                {"count_reads.bam", null, 8L},
-                {"count_reads.cram", "count_reads.fasta", 8L},
-                {"count_reads_sorted.sam", null, 8L},
-                {"count_reads_sorted.bam", null, 8L},
-                {"count_reads_sorted.cram", "count_reads.fasta", 8L},
+                {"", "count_reads.sam", null, 8L},
+                {"", "count_reads.bam", null, 8L},
+                {"symbolic_name:", "count_reads.bam", null, 8L},
+                {"", "count_reads.cram", "count_reads.fasta", 8L},
+                {"", "count_reads_sorted.sam", null, 8L},
+                {"", "count_reads_sorted.bam", null, 8L},
+                {"", "count_reads_sorted.cram", "count_reads.fasta", 8L},
         };
     }
 
     @Test(dataProvider = "filenames", groups = "spark")
-    public void testCountReads(final String fileIn, final String referenceName, final long expectedCount) throws Exception {
+    public void testCountReads(final String prefix, final String fileIn, final String referenceName, final long expectedCount) throws Exception {
         final File ORIG_BAM = new File(getTestDataDir(), fileIn);
         final File outputTxt = createTempFile("count_reads", ".txt");
 
         final ArgumentsBuilder args = new ArgumentsBuilder();
-        args.addInput(ORIG_BAM);
+        args.add("--input");
+        args.add(prefix + ORIG_BAM.getAbsolutePath());
```

Comment:
Test that you can access the read inputs via symbolic name within a Spark tool (again, using a toy minimal `GATKSparkTool` that does asserts)